app/bundles/ReportBundle/Controller/ReportController.php

```diff
@@ -627,6 +627,32 @@ public function viewAction(Request $request, $objectId, $reportPage = 1)
             $filterSettings[$filterDefinitions->definitions[$filter['column']]['alias']] = $filter['value'];
         }
 
+        foreach ($entity->getFilters() as $filter) {
+            if (!isset($filter['dynamic']) || 1 !== $filter['dynamic']) {
+                continue;
+            }
+
+            $column     = $filter['column'] ?? null;
+            $definition = (null !== $column) ? ($filterDefinitions->definitions[$column] ?? []) : [];
+            $alias      = $definition['alias'] ?? null;
+
+            if (null === $alias || isset($filterSettings[$alias])) {
+                continue;
+            }
+
+            $value = $filter['value'] ?? null;
+            if ('' === $value || null === $value) {
```

Comment:
```suggestion
            if (!empty($value)) {
```
... but only if 0 is not considered a valid `$value`'s value in this context.

Reply:
Users might effectively want to filter by 0 depending on the custom field, so I'd play safe here